### DISCUSSION TITLE
Не дает авторизоваться, не качается манга

### DISCUSSION THREAD

**Discussion by Mickael6Whelan** [lirix360/ReadmangaGrabber#79] [Q&A]
Сделала все, уже 3 часа сижу, пытаюсь запустить прогу. И куки скачала, и файл переименовала. Один раз получилось скачать, но были пустые папки... Он будто не видит, как я захожу и логинюсь на сайте. Хэлп :(

**Comment by uchihaamaya** [comment]
Та же ситуация. Делаю по инструкции: открываю браузер, вхожу на сайт манги (предварительно выхожу из профиля), ввожу логин и пароль, сохраняю файл cookie через cookies.txt в ту же папку, где лежит grabber_win_x64.exe, переименовываю в mintmanga.txt. Запускаю grabber_win_x64.exe, ввожу адрес манги, скачиваю главы и возникает ошибка "Для скачивания указанной манги необходимо авторизаваться на сайте!". Что делаю не так? @Mickael6Whelan , у вас получилось?

**Comment by worldbe4you** [comment]
Аналогичная проблема при скачивании с минтманги... Танцы с бубном (инструкция по авторизации) не помогают((

**Comment by lirix360** [comment]
@uchihaamaya @worldbe4you Сейчас проверил - нормально качает с минтманги. Можно адреса манги которая не скачивается?

**Reply by worldbe4you** [reply]
Первая попавшаяся скачалась (https://mintmanga.live/devushka_u_moria), а желаемое - нет (https://mintmanga.live/neobiatnyi_okean__A35c96). В чем может быть причина?

**Reply by lirix360** [reply]
Сейчас без проблем скачал пару глав из https://mintmanga.live/neobiatnyi_okean__A35c96

**Reply by uchihaamaya** [reply]
Например, https://mintmanga.live/zolotoi_grei__A5327. 
Явно что-то не так с действиями, но не могу понять что именно. Разрешения cookie для https://mintmanga.live стоит

**Reply by uchihaamaya** [reply]
А вот https://readmanga.live/vyjit_v_kachestve_jeny_geroia__A5664 скачивается нормально, хотя файла cookie для него нет. https://mintmanga.live/strannaia_magiia_chisla_9_2022 тоже скачивается. А вот https://mintmanga.live/zolotoi_grei__A5327  все еще нет/ https://mintmanga.live/neobiatnyi_okean__A35c96 тоже не могу скачать

**Reply by uchihaamaya** [reply]
Ура, кажется причина найдена. Переименовала файл как по инструкции в mintmanga.txt. Но судя по всему в инструкции уже был включен формат файла. То есть переименовывать файл нужно в "mintmanga", txt - это формат))) @worldbe4you, попробуйте, у меня начало скачиваться, вдруг поможет

**Reply by worldbe4you** [reply]
@uchihaamaya к сожалению, причину почему у меня не работает я так и не обнаружила. Никакие переименования не помогают, смена браузера и компа тоже)

**Reply by worldbe4you** [reply]
@lirix360 @uchihaamaya проблему устранила сменой браузера на firefox. Ни хром, ни яндекс работать не стали...
